KarhooUISDK/Screens/BookingRequestScreen/BookingRequest/SupplierView/SupplierView.swift

```diff
@@ -80,19 +89,25 @@ final class SupplierView: UIView {
                                                     constant: 0)].map { $0.isActive = true }
         
         _ = [vehicleType.leadingAnchor.constraint(equalTo: supplierName.leadingAnchor),
-             vehicleType.topAnchor.constraint(equalTo: supplierName.bottomAnchor, constant: 3.0),
-             vehicleType.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -5.0)].map { $0.isActive = true }
+             vehicleType.topAnchor.constraint(equalTo: supplierName.bottomAnchor, constant: 3.0)].map { $0.isActive = true }
         
         _ = [vehicleCapacityView.centerYAnchor.constraint(equalTo: vehicleType.centerYAnchor),
              vehicleCapacityView.leadingAnchor.constraint(equalTo: vehicleType.trailingAnchor,
                                                           constant: 13.0)].map { $0.isActive = true }
+
+        cancellationInfo.topAnchor.constraint(equalTo: vehicleType.bottomAnchor, constant: 3.0).isActive = true
+        cancellationInfo.leadingAnchor.constraint(equalTo: vehicleType.leadingAnchor).isActive = true
+        cancellationInfo.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
+        cancellationInfo.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 5.0).isActive = true
     }
     
     func set(viewModel: QuoteViewModel) {
         supplierName.text = viewModel.fleetName
         let vehicleTypeText = viewModel.showPickUpLabel ? viewModel.carType + " | " + viewModel.pickUpType :
                               viewModel.carType
         vehicleType.text = vehicleTypeText
+        cancellationInfo.text = String(format: UITexts.Quotes.freeCancellation, "\(viewModel.freeCancellationMinutes)")
```

Comment:
Perhaps we can move this formatting to the QuoteViewModel so we don't need to repeat it across three places